### ISSUE TITLE
Crashes without printing any error!

### ISSUE THREAD

**Issue by shantanu-kumar242** [microsoft/IRNet#26]
When I run the train.sh file then after 2-3 minutes without giving any error it exits. I have sufficient amount (16GB) of RAM. I have attached the screenshot. Does anyone faced the same issue? How can it be resolved? Tried on colab as well as local machine. Same issue is there on both.
Please help.
![issue](https://user-images.githubusercontent.com/51315728/75883092-71e96980-5e48-11ea-9ff9-f1a36ac8320d.png)

**Comment by jaydeepb-inexture** [comment]
@shantanu-kumar242 try with adding gpu_id and a name for folder
sh train.sh 0[GPU_ID] model_result[directory_name]
this directory will be created in the saved_model folder in which you have stored pretrained model as they have mentioned in READ.md file.
if you want to see the errors than you can find it as directory_name.log if you have mentioned directory_name as i have said above.

**Comment by shantanu-kumar242** [comment]
> @shantanu-kumar242 try with adding gpu_id and a name for folder
> sh train.sh 0[GPU_ID] model_result[directory_name]
> this directory will be created in the saved_model folder in which you have stored pretrained model as they have mentioned in READ.md file.
> if you want to see the errors than you can find it as directory_name.log if you have mentioned directory_name as i have said above.

Thanks for help.
I went through log file and found the following error:
![issue1](https://user-images.githubusercontent.com/51315728/75966119-d7446580-5eef-11ea-9806-20f73dbc9bbb.png)
I ran the command as mentioned: conda install pytorch torchvision cudatoolkit=9.2 -c pytorch -c defaults -c numba/label/dev but still I am getting the same error. I am running this model on cpu as my system doesn't have gpu. Plz help

**Comment by jaydeepb-inexture** [comment]
which system are you using?

**Comment by shantanu-kumar242** [comment]
> which system are you using?

Linux

**Comment by jaydeepb-inexture** [comment]
https://pytorch.org/
go here find approriate option for your system and install pytorch
select none in cuda as you have no gpu.

**Comment by shantanu-kumar242** [comment]
Thanks a lot now it's working.
